web/client/plugins/Widgets.jsx

```diff
@@ -279,13 +276,15 @@ compose(
 
 class Widgets extends React.Component {
     static propTypes = {
-        enabled: PropTypes.bool
+        enabled: PropTypes.bool,
+        enableZoomInTblWidget: PropTypes.bool
     };
     static defaultProps = {
-        enabled: true
+        enabled: true,
+        enableZoomInTblWidget: true
     };
     render() {
-        return this.props.enabled ? <WidgetsView {...this.props /* pass options to the plugin */ } /> : null;
+        return this.props.enabled ? <WidgetsView enableZoomInTblWidgetInMapViewer={this.props.enableZoomInTblWidget} {...this.props /* pass options to the plugin */ } /> : null;
```

Comment:
hi, can't you sue the same name as in the cfg? 
so I would remove "enableZoomInTblWidgetInMapViewer={this.props.enableZoomInTblWidget}" it since with the spreading you do after you will anyway override the prop.